lib/_stream_readable.js

```diff
@@ -680,16 +680,17 @@ Readable.prototype.pipe = function(dest, pipeOpts) {
   // on the source.  This would be more elegant with a .once()
   // handler in flow(), but adding and removing repeatedly is
   // too slow.
-  const ondrain = pipeOnDrain(src);
-  dest.on('drain', ondrain);
+  let ondrain;
```

Comment:
The comment above would need to be adjusted.

Reply:
Moved